pyqt-apps/scripts/sirius-hla-as-di-scrn.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python-sirius
+
+"""Open Window of Specified Screen."""
+
+import sys
+import argparse as _argparse
+from siriuspy.envars import vaca_prefix
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QWidget, QLabel, QSpacerItem, \
+                           QSizePolicy as QSzPlcy, QGridLayout
+from pydm.widgets import PyDMEnumComboBox, PyDMLabel
+from siriushla.sirius_application import SiriusApplication
+from siriushla.widgets import SiriusScrnView, PyDMLed
+from siriushla import util
+from siriushla.widgets.windows import SiriusMainWindow
+
+
+parser = _argparse.ArgumentParser(
+    description="Run Interface of Specified Screen.")
+parser.add_argument('scrn', type=str, default='TB-01:DI-Scrn-1',
+                    help='Select Screen.')
+parser.add_argument('-p', "--prefix", type=str, default=vaca_prefix,
+                    help="Define the prefix for the PVs in the window.")
+args = parser.parse_args()
+
+scrn_device = args.scrn
+prefix = args.prefix
+
+app = SiriusApplication()
+util.set_style(app)
+
+cw = QWidget()
+scrn_view = SiriusScrnView(prefix='', device=scrn_device)
+cb_scrntype = PyDMEnumComboBox(
+    parent=cw, init_channel='ca://'+prefix+scrn_device+':ScrnType-Sel')
```

Comment:
@anacso17, I think you can remove the `'ca://'` from this new file. You can leave the other `'ca://'` present in the modified files because I have already removed them in my PR.